_project-shortlist/2013/2013-template.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
This file should be removed from the repo. It's a duplicate of the template at the root of project-shortlist

Reply:
I was creating templates within each year folder with the specific year at the top and then deleting them and recreating them for the next year as I go along. I would prefer to leave for now as I will use at as soon as I start on 2013 files but can remove it if I need to.

Reply:
If that's the case, be sure to change `published` to `false`. Since this is a template it should never be published, even if it will be deleted in the future.